### PR TITLE
feat-US34012: Add proposal Status Stepper to Proposal Details

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**
Screenshot 2023-02-02 at 4 25 19 PM
Screenshot 2023-02-02 at 4 25 37 PM
Screenshot 2023-02-02 at 4 25 48 PM
Screenshot 2023-02-02 at 4 26 01 PM
Screenshot 2023-02-02 at 4 27 32 PM

### CODE REVIEW

File: src/dex-ui-components/base/Stepper/Stepper.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import { Stepper, Step, StepLabel } from "@material-ui/core";
+import { CheckCircleIcon, createIcon } from "@chakra-ui/icons";
+import { Color } from "../../themes";
+import { Flex } from "@chakra-ui/layout";
+import { Box, Text } from "@chakra-ui/react";
+import { StepperStyles } from "./StepperStyles";
+import { StepperState, ProposalStateIcon } from "./types";
+
+const DisabledIcon = createIcon({
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui-components/base/Stepper/Stepper.tsx:9]: We can leave these Icons here for now but we may want to move them into the dex-ui Icons folder once we create the official DEX Component Library.

**Comment by damiano-melcarne-lab49** [comment]
@roshan-bisht-lab49 The code looks good but the stepper UI needs to match the wireframes in terms of size, padding, etc.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]
@roshan-bisht-lab49 The code looks good but the stepper UI needs to match the wireframes in terms of size, padding, etc.

File: src/dex-ui-components/base/Stepper/Stepper.tsx (hunk 1 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui-components/base/Stepper/Stepper.tsx]: Done, I have moved the icons under dex-ui-components/base/Icons folder.

Comment (roshan-bisht-lab49) [reply src/dex-ui-components/base/Stepper/Stepper.tsx]: Also the reduced the size of `Icons` to match the Wireframe

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Pull request merged**